--- a/luci-mod-status/htdocs/luci-static/resources/view/status/index.js
+++ b/luci-mod-status/htdocs/luci-static/resources/view/status/index.js
@@ -2,6 +2,7 @@
 'require view';
 'require dom';
 'require poll';
+'require fs';
 'require network';
 'require ui';
 
@@ -278,26 +279,14 @@ return view.extend({
 	},
 
 	load: function() {
-		var includeModules = [
-			{ name: 'view.status.include.10_system' },
-			{ name: 'view.status.include.15_ports', title: _('Port status'), deferFirstLoad: true },
-			{ name: 'view.status.include.20_memory' },
-			{ name: 'view.status.include.25_storage' },
-			{ name: 'view.status.include.30_network' },
-			{ name: 'view.status.include.40_dhcp', title: _('Online Hosts'), deferFirstLoad: true },
-			{ name: 'view.status.include.50_dsl' },
-			{ name: 'view.status.include.60_wifi' }
-		];
-
-		return Promise.all(includeModules.map(L.bind(function(spec) {
-			if (spec.deferFirstLoad)
-				return this.wrapDeferredInclude(spec);
-
-			return L.resolveDefault(L.require(spec.name), null);
-		}, this))).then(function(includes) {
-			return includes.filter(function(include) {
-				return include != null;
-			});
+		return L.resolveDefault(fs.list('/www' + L.resource('view/status/include')), []).then(function(entries) {
+			return Promise.all(entries.filter(function(e) {
+				return (e.type == 'file' && e.name.match(/\.js$/));
+			}).map(function(e) {
+				return 'view.status.include.' + e.name.replace(/\.js$/, '');
+			}).sort().map(function(n) {
+				return L.require(n);
+			}));
 		});
 	},


--- a/luci-mod-status/htdocs/luci-static/resources/view/status/include/20_memory.js
+++ b/luci-mod-status/htdocs/luci-static/resources/view/status/include/20_memory.js
@@ -36,14 +36,14 @@ return baseclass.extend({
 			available = Math.min(mem.total ?? Infinity, Math.max(0, available + (mem.buffered || 0)));
 
 		var fields = [
-			_('Total Available'), available, mem.total,
+			_('Used'), (mem.total && mem.available) ? (mem.total - available): null, mem.total,
 		];
 
 		if (mem.cached)
 			fields.push(_('Cached'), mem.cached, mem.total);
 
 		if (swap.total > 0)
-			fields.push(_('Swap free'), swap.free, swap.total);
+			fields.push(_('Swap used'), swap.total - swap.free, swap.total);
 
 		var table = E('table', { 'class': 'table' });
 

--- a/luci-mod-status/htdocs/luci-static/resources/view/status/include/10_system.js
+++ b/luci-mod-status/htdocs/luci-static/resources/view/status/include/10_system.js
@@ -36,7 +36,7 @@ var callCPUInfo = rpc.declare({
 
 var callCPUUsage = rpc.declare({
 	object: 'luci',
-	method: 'getCPUUsage'
+	method: 'getTempInfo'
 });
 
 return baseclass.extend({
@@ -85,15 +85,9 @@ return baseclass.extend({
 
 		var fields = [
 			_('Hostname'),         boardinfo.hostname,
-			_('Model'),            boardinfo.model + cpubench.cpubench,
-			_('Architecture'),     cpuinfo.cpuinfo || boardinfo.system,
+			_('Architecture'),     (cpuinfo.cpuinfo || boardinfo.system) + ' ' + cpubench.cpubench,
 			_('Target Platform'),  (L.isObject(boardinfo.release) ? boardinfo.release.target : ''),
-			_('Firmware Version'), (L.isObject(boardinfo.release)
-				? '%s%s / '.format(
-					boardinfo.release.description || '',
-					boardinfo.release.revision ? boardinfo.release.revision : ''
-				)
-				: '') + (luciversion || ''),
+			_('Firmware Version'), (L.isObject(boardinfo.release) ? boardinfo.release.description + ' / ' : '') + (luciversion || ''),
 			_('Kernel Version'),   boardinfo.kernel,
 			_('Local Time'),       datestr,
 			_('Uptime'),           systeminfo.uptime ? '%t'.format(systeminfo.uptime) : null,
@@ -101,10 +95,18 @@ return baseclass.extend({
 				systeminfo.load[0] / 65535.0,
 				systeminfo.load[1] / 65535.0,
 				systeminfo.load[2] / 65535.0
-			) : null,
-			_('CPU usage (%)'),    cpuusage.cpuusage
+			) : null
 		];
 
+		if (cpuusage.tempinfo) {
+			fields.splice(6, 0, _('Temperature'));
+			fields.splice(7, 0, cpuusage.tempinfo);
+		}
+		if (boardinfo.model != "Default string Default string") {
+			fields.splice(2, 0, _('Model'));
+			fields.splice(3, 0, boardinfo.model);
+		}
+
 		var table = E('table', { 'class': 'table' });
 
 		for (var i = 0; i < fields.length; i += 2) {

--- a/luci-mod-status/htdocs/luci-static/resources/view/status/nftables.js
+++ b/luci-mod-status/htdocs/luci-static/resources/view/status/nftables.js
@@ -675,7 +675,6 @@ return view.extend({
 	checkLegacyRules: function(ipt4save, ipt6save) {
 		if (ipt4save.match(/\n-A /) || ipt6save.match(/\n-A /)) {
 			ui.addNotification(_('Legacy rules detected'), [
-				E('p', _('There are legacy iptables rules present on the system. Mixing iptables and nftables rules is discouraged and may lead to incomplete traffic filtering.')),
 				E('button', {
 					'class': 'btn cbi-button',
 					'click': function() { location.href = 'nftables/iptables' }
diff --git a/luci-mod-status/htdocs/luci-static/resources/view/status/include/15_ports.js b/luci-mod-status/htdocs/luci-static/resources/view/status/include/26_ports.js
similarity index 100%
rename from luci-mod-status/htdocs/luci-static/resources/view/status/include/15_ports.js
rename to luci-mod-status/htdocs/luci-static/resources/view/status/include/26_ports.js
