--- a/qBittorrent-Enhanced-Edition/files/qbittorrent.init
+++ b/qBittorrent-Enhanced-Edition/files/qbittorrent.init
@@ -49,6 +49,7 @@ start_service() {
 		Cookies=@Invalid()
 
 		[Preferences]
+		General\Locale=zh_CN
 		WebUI\CSRFProtection=false
 		WebUI\Password_PBKDF2="@ByteArray(zH92gnE9xijRN5IjzIPU+A==:+JxpKBWsSyuzpm/9LrhO2uLQSGsBS5giqF0AYRU8COcMXn5AIaSsL2S9hIuB20wkhsMfEoN+77Q9BmAd3ysYxw==)"
 		WebUI\Port=$http_port

--- a/qbittorrent/files/qbittorrent.init
+++ b/qbittorrent/files/qbittorrent.init
@@ -49,6 +49,7 @@ start_service() {
 		Cookies=@Invalid()
 
 		[Preferences]
+		General\Locale=zh_CN
 		WebUI\CSRFProtection=false
 		WebUI\Password_PBKDF2="@ByteArray(zH92gnE9xijRN5IjzIPU+A==:+JxpKBWsSyuzpm/9LrhO2uLQSGsBS5giqF0AYRU8COcMXn5AIaSsL2S9hIuB20wkhsMfEoN+77Q9BmAd3ysYxw==)"
 		WebUI\Port=$http_port
